community-packs.json

```diff
@@ -1,2 +1,10 @@
 {
+    "messenger_track_pack": {
+        "name": "The Messenger Randomizer Track Pack",
+        "author": "alwaysintreble",
+        "platform": "PC",
+        "versions_url": "https://github.com/alwaysintreble/TheMessengerTrackPack/versions.json",
+        "icon_url": "https://github.com/alwaysintreble/TheMessengerTrackPack/blob/master/The%20Messenger%20Track%20Pack/images/SaveIcon.png",
```

Comment:
neither of those links work

Reply:
Oops. Fixed them.

Reply:
That's actually still not correct :sweat_smile: We need the raw/direct url (i.e.https://raw.githubusercontent.com/... )

Reply:
ah ok i see lol